git-attributes/README.md

```diff
@@ -35,26 +35,46 @@ are for GNU/Linux platforms and Mac.
    to an interpreter. GNU/Linux systems will have a problem running them, e.g., `bash\r` is not
    recognized as an interpreter.
 
-5. Git is mostly suitable for text files, but it can also handle binary files
-   and you can set programs to show the diff. You need to have installed exif (or exiftool)
-   for this to work. If you don't have a png image on your machine, you can use:
-   https://www.freepngimg.com/download/mario/20698-7-mario-transparent-background.png
+   Note: The line endings can also be controlled through
+   [`git config`](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration) but that is a whole different topic.
```

Comment:
"... but that is a different topic."

Reply:
Thanks for finding this. Fixed in new commit.